examples/benchmark/benchmark.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+num_of_processes=$1
+eval_time=$2
+packet_size=$3
+bandwidth=$4
+platform=$5
+
```

Comment:
Please implementI  `-h` / `--help` for the tl;dr folks.

Reply:
Thanks for the idea, your suggestion made me change the positional parameters to flags and set default values. This makes the whole script more convenient to use IMO.